torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1615,14 +1615,13 @@ at::Tensor XLANativeFunctions::index_add(const at::Tensor& self, int64_t dim,
       bridge::GetXlaTensor(source), alpha));
 }
 
-at::Tensor& XLANativeFunctions::index_copy_(at::Tensor& self, int64_t dim,
+at::Tensor& XLANativeFunctions::index_copy(const at::Tensor& self, int64_t dim,
                                             const at::Tensor& index,
                                             const at::Tensor& source) {
   XLA_FN_COUNTER("xla::");
   XLATensor self_tensor = bridge::GetXlaTensor(self);
-  XLATensor::index_copy_(self_tensor, dim, bridge::GetXlaTensor(index),
```

Comment:
nit: you can probably kill the code for `XLATensor::index_copy_()` too?

Reply:
@bdhirsh I believe that's what I did. This change no longer has the call to `torch_copy_`. Is the way it's done incorrect for you?

Reply:
I just meant that there's also a `index_copy_()` static method on the `XLATensor` class (in `torch_xla/csrc/tensor.h` that should be unused now. So I think we can remove that method too?

Reply:
That's correct. I understand what you meant. 
FYI: I am fixing a local build issue in my dev env. Will push the final changes soon.